src/index.js

```diff
@@ -1,33 +1,34 @@
 // import _ from 'lodash';
 import './style.css';
 
-const todoTasks = [
-  {
-    index: 0,
-    description: 'Edit tiktok video',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'send draft to client',
-    completed: true,
-  },
-];
+import addNewTask from './modules/add-task.js';
+import deleteTask from './modules/delete-task.js';
+import editTask from './modules/edit-task.js';
+
+let todoTasks = [];
+
+// class TodoTasks {
+//   constructor(description, completed, index) {
+//     this.description = description;
+//     this.completed = completed;
+//     this.index = index;
+//   }
+// }
```

Comment:
- [ ] Please remove the commented code from the files, It's not good practice to leave commented code.